### PR TITLE
fix: revert changes to key manager branch seed strings

### PR DESCRIPTION
**Pull request by philipr-za**
Description
---
During the Key Manager refactor the strings used as branch seeds by the key manager were changed. This broke recovery as the keys generated for the rewind process were wrong and the rewound key index could no longer be found.

### CODE REVIEW

File: base_layer/wallet/src/output_manager_service/resources.rs [hunk 1]
```diff
@@ -59,24 +57,25 @@ pub enum OutputManagerKeyManagerBranch {
...
+impl OutputManagerKeyManagerBranch {
+    pub fn get_branch_key(&self) -> String {
+        match self {
+            OutputManagerKeyManagerBranch::Spend => "".to_string(),
+            OutputManagerKeyManagerBranch::SpendScript => "script".to_string(),
+            OutputManagerKeyManagerBranch::Coinbase => "coinbase".to_string(),
+            OutputManagerKeyManagerBranch::CoinbaseScript => "coinbase_script".to_string(),
+            OutputManagerKeyManagerBranch::RecoveryViewOnly => "recovery_viewonly".to_string(),
+            OutputManagerKeyManagerBranch::RecoveryBlinding => "recovery_blinding".to_string(),
+            OutputManagerKeyManagerBranch::RecoveryByte => "Recovery_byte".to_string(),
```
Comment (sdbondi) [inline base_layer/wallet/src/output_manager_service/resources.rs:71]: This doesn't look like it's been reverted

Comment (philipr-za) [reply base_layer/wallet/src/output_manager_service/resources.rs:71]: yup that one is supposed to stay that way

Comment (sdbondi) [reply base_layer/wallet/src/output_manager_service/resources.rs:71]: Ah probably wasn't there before and needs to stay the same now

Comment (sdbondi) [reply base_layer/wallet/src/output_manager_service/resources.rs:71]: lol thanks github didnt refresh

File: base_layer/wallet/src/output_manager_service/resources.rs [hunk 2]
```diff
@@ -59,24 +57,25 @@ pub enum OutputManagerKeyManagerBranch {
...
+        }
     }
 }
 
 #[allow(clippy::from_over_into)]
 impl Into<String> for OutputManagerKeyManagerBranch {
```
Comment (sdbondi) [inline base_layer/wallet/src/output_manager_service/resources.rs]: I would remove this impl, Clippy is right about it

File: base_layer/wallet/src/key_manager_service/storage/sqlite_db/mod.rs [hunk 3]
```diff
@@ -94,11 +94,11 @@ impl KeyManagerSqliteDatabase {
         if !old_state.is_empty() {
             // there should only be 1 if there is an old state.
             let spending_km = KeyManagerState {
-                branch_seed: OutputManagerKeyManagerBranch::Spend.to_string(),
+                branch_seed: OutputManagerKeyManagerBranch::Spend.into(),
```
Comment (sdbondi) [inline base_layer/wallet/src/key_manager_service/storage/sqlite_db/mod.rs]: Should be `get_branch_key`?

File: base_layer/wallet/src/output_manager_service/resources.rs [hunk 4]
```diff
@@ -59,24 +57,25 @@ pub enum OutputManagerKeyManagerBranch {
...
-            OutputManagerKeyManagerBranch::Spend => "Spend",
-            OutputManagerKeyManagerBranch::SpendScript => "Script",
-            OutputManagerKeyManagerBranch::Coinbase => "Coinbase",
-            OutputManagerKeyManagerBranch::CoinbaseScript => "Coinbase_script",
-            OutputManagerKeyManagerBranch::RecoveryViewOnly => "Recovery_viewonly",
-            OutputManagerKeyManagerBranch::RecoveryBlinding => "Recovery_blinding",
-            OutputManagerKeyManagerBranch::RecoveryByte => "Recovery_byte",
-        };
-        fmt.write_str(response)
+/// Warning: Changing these strings will affect the backwards compatibility of the wallet with older databases or
```
Comment (sdbondi) [inline base_layer/wallet/src/output_manager_service/resources.rs]: This probably shouldnt be a docstring, or should be moved above the method.

**Review by stringhandler — DISMISSED** [review]
Much better

**Review by delta1 — APPROVED** [review]

**Pull request merged**